hack/build/docker/cdi-func-test-file-host-http/Dockerfile

```diff
@@ -1,10 +1,17 @@
-FROM fedora:28
+FROM fedora:29
 
 MAINTAINER "The KubeVirt Project" <kubevirt-dev@googlegroups.com>
 
 ENV container docker
 
-RUN dnf -y update && dnf -y install nginx && dnf clean all -y
+COPY fedora.repo /tmp/fedora_ci.dnf.conf
+
+RUN cat /tmp/fedora_ci.dnf.conf && \
```

Comment:
is `cat` command needed?

Reply:
No, I just used it to see what the content of the file was so I knew the right one was being copied into the container. I will remove now that I am satisfied it is correct.